Add the defaultActiveTabKey property for the Card component

u3u opened the pull request:
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [x] Make sure that you propose PR to right branch: bugfix for `master`, feature for latest active branch `feature-x.x`.
* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [x] Update API docs for the component.
  * [x] Update/Add demo to demonstrate new feature.
  * [x] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

u3u committed:
Add the defaultActiveTabKey property for the Card component (close #8789, #8942)

afc163 commented on components/card/demo/tabs.md:
```diff
@@ -71,6 +71,7 @@ class TabsCard extends React.Component {
         <Card
           style={{ width: '100%' }}
           tabList={tabListNoTitle}
+          defaultActiveTabKey={this.state.noTitleKey}
```
It should be `activeKey` if you assign a state value to it.

u3u replied:
Oh, yes, I changed again

afc163 committed:
Fix large tabs font size, close #9509

u3u committed:
`activeTabKey` should be added

yesmeck committed:
docs: Add TreeSelect[dropdownClassName]

yesmeck committed:
Fix passing dropdownClassName to tree-select

dependabot[bot] committed:
build: update remark-parse requirement to ^5.0.0 (#9545)

Updates the requirements on [remark-parse](https://github.com/remarkjs/remark) to permit the latest version.
- [Release notes](https://github.com/remarkjs/remark/releases)
- [Commits](https://github.com/remarkjs/remark/commits/remark@5.0.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

chenshuai2144 committed:
add Tooltip  contextMenu doc

yesmeck committed:
Improve Grid and Layout type definition

RaoHai committed:
fix: focus editor (#9548)

afc163 committed:
Fix test case for new jsdom (#9527)

* Fix test case for new jsdom

* use setTimeout as raf in jest jsdom

* Fix cancelAnimationFrame

* Add comment for facebook/jest#5147

* longer timeout

* fix snap

* upgrade antd-tools

dependabot[bot] committed:
Update typescript requirement to ~2.7.2 (#9522)

Updates the requirements on [typescript](https://github.com/Microsoft/TypeScript) to permit the latest version.
- [Release notes](https://github.com/Microsoft/TypeScript/releases)
- [Commits](https://github.com/Microsoft/TypeScript/commits/v2.7.2)

Signed-off-by: dependabot[bot] <support@dependabot.com>

afc163 committed:
Fix a ts error

dependabot[bot] committed:
build: update react-slick requirement to ~0.20.0 (#9543)

Updates the requirements on [react-slick](https://github.com/akiran/react-slick) to permit the latest version.
- [Changelog](https://github.com/akiran/react-slick/blob/master/CHANGELOG.md)
- [Commits](https://github.com/akiran/react-slick/commits)

Signed-off-by: dependabot[bot] <support@dependabot.com>

afc163 committed:
Fix test cases

afc163 committed:
site: fix intersection-observer polyfill

afc163 committed:
docs: update recommendation

pierre-H committed:
Fix typo WeexPickerProps -> WeekPickerProps (#9564)

afc163 committed:
use lodash

idabagusangga committed:
Fixed typo on Visualization rules (#9575)

Style of a navigation should conform to the its level.

should be

Style of a navigation should conform to its level.

yesmeck committed:
Improve Radio/Checkbox type definition

Close #9574

yesmeck committed:
Remove AbstractCheckboxChangeEvent, fix TS4029 error

See https://github.com/Microsoft/TypeScript/issues/9944

KevinBon committed:
Update index.en-US.md (#9579)

muzea committed:
add transitionName from message.config (#9580)

* add transitionName from message.config

* Update index.en-US.md (#9579)

* modify doc

dependabot[bot] committed:
build: update react-virtualized requirement to ~9.18.5 (#9544)

Updates the requirements on [react-virtualized](https://github.com/bvaughn/react-virtualized) to permit the latest version.
- [Changelog](https://github.com/bvaughn/react-virtualized/blob/master/CHANGELOG.md)
- [Commits](https://github.com/bvaughn/react-virtualized/commits/9.18.5)

Signed-off-by: dependabot[bot] <support@dependabot.com>

valleykid committed:
When treeNode is disabled, its switcher is highlight and clickabled (#9539)

* When treeNode is disabled, its switcher is highlight and clickabled

* rc-tree@1.7.11

yesmeck committed:
Fix moment require (#9528)

Fix #9502

afc163 commented on components/card/index.tsx:
```diff
@@ -160,9 +162,19 @@ export default class Card extends React.Component<CardProps, {}> {
 
+    const extraProps = {
+      [activeTabKey ? 'activeKey' : 'defaultActiveKey']:
+        activeTabKey || defaultActiveTabKey,
```
What about `activeTabKey={0}`?

u3u replied:
Sorry, I didn't understand what you meant.
The `activeTabKey` should be passed in a string type.
So how should `activeTabKey={0}` be handled?

afc163 replied:
Use `activeTabKey === undefined` is more robust in case of  `activeTabKey=""` or else.

u3u replied:
Thank you for pointing, updated.

u3u committed:
Improve

u3u committed:
Update snapshot

u3u commented:
I'm very sorry, I accidentally got the merged changes into this PR! Will it cause trouble to you? I just want to update the snapshot, otherwise CI has been wrong.
